pkg/logs/config/source.go

```diff
@@ -60,7 +57,7 @@ func NewLogSource(name string, config *LogsConfig) *LogSource {
 		inputs:           make(map[string]bool),
 		lock:             &sync.Mutex{},
 		Messages:         NewMessages(),
-		BytesRead:        expvar.Int{},
```

Comment:
Why are we not using `*expvar.Int{}` as shown in `atomics.md` ? Are those 2 things equivalent ?

Reply:
expvar.Int{} is appropriate if the value is used as an expvar -- but this one isn't.

Reply:
(mentioned in the commit message: `This value was of type `expvar.Int{}` but never used as an expvar.`)

Reply:
Didn't see the message mb, thanks !